src/components/ResultsPage/ResultsPage.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import './ResultsPage.css'
+import NavBar from '../NavBar/NavBar'
+import ResultsSearchSection from './ResultsSearchSection/ResultsSearchSection'
+import ResultsBody from './ResultsBody/ResultsBody'
+import FiltersBody from './FiltersBody/FiltersBody'
+
+function ResultsPage() {
+    
+    
+    //TODO: When the search functionality is properly in place, 
+    //use a ternary to display a specific message if no search
+    //results were found
+    return (
+        <div>
+            <NavBar />
+            <ResultsSearchSection />
+            <div className="results-page-section">
+                <div className="results-body-section"><ResultsBody /></div>
```

Comment:
In general, we shouldn't surround single components with a styled div. Instead, you can move the div to inside of the React component and style it within the css file for that component! Ask me if you're not sure what this means.